packages/cli/src/tools/config/__tests__/index.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * @flow
+ */
+
+import loadConfig from '../';
+
+import {
+  cleanup,
+  writeFiles,
+  getTempDirectory,
+} from '../../../../../../e2e/helpers';
```

Comment:
@thymikee this is not an e2e test, but these utilities are useful. Shall we move them to `testUtils` or something to clarify the naming?

Reply:
I'd like to keep it in e2e as it implies working with real FS

Reply:
Ok, let's leave it as is for now.